src/applications/disability-benefits/all-claims/pages/privateMedicalRecords.js

```diff
@@ -78,6 +79,26 @@ export const uiSchema = {
         _.get(DATA_PATHS.hasPrivateRecordsToUpload, data, false),
     },
   ),
+  'view:patientAcknowledgement': {
+    'ui:title': ' ',
+    'ui:help': patientAcknowledgmentText,
+    'ui:options': {
+      expandUnder: 'view:uploadPrivateRecordsQualifier',
+      expandUnderCondition: data =>
+        _.get('view:hasPrivateRecordsToUpload', data, false) === false,
+      showFieldLabel: true,
+    },
+    'view:acknowledgement': {
+      'ui:title': 'Patient Acknowledgement',
```

Comment:
Would this make more sense if it read something like "I have read and accept the patient acknowledgment below"?

I'm no content editor, though--it'll still need reviewed by somebody who knows what they're doing. :stuck_out_tongue: 

Like I said above, it's okay as a placeholder for now.

Reply:
@Jennywalter passing this not along to you as requested as well!

Reply:
Hi @cvalarida this was approved by Paul Shute and Andrea Schneider during the requirements process, so it will need to stay as it is.  Thanks!